server-side/framework/python/django/docker-compose.stage.yml

```diff
@@ -0,0 +1,36 @@
+version: "3.7"
+services:
+  db:
+    image: mdillon/postgis:10
+    restart: always
```

Comment:
I like this on a production docker-compose file nice work

Reply:
Cool, thanks